src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/MissingJavadocTypeCheck.java

```diff
@@ -53,7 +53,8 @@
  * </li>
  * <li>
  * Property {@code skipAnnotations} - specify annotations that allow missed
- * documentation. Only short names are allowed, e.g. {@code Generated}.
+ * documentation. Both short(e.g. {@code Generated}) and qualified(e.g. {@code MyClass.Annotation})
```

Comment:
We do not fully support qualified names.

We simply match name as is. If user use two ways of same annotation (short name and with  firs package) he needs to use both forms in config if do not want violation.

Not sure how to word it properly.

Reply:
I think simply say as things are.?

> ..missed documentation. Specified annotations are matched as is - `Annotation` will only match `@Annotation` but not `@MyClass.Annotation`.

Reply:
`...But not any qualified form of it`

Reminder:Fully qualified name is name with package.

Reply:
That sounds better. Just to make sure before I make the changes.

>..missed documentation. Specified annotations are matched as is - Annotation will only match `@Annotation` but not any qualified form of it(e.g. `@MyClass.Annotation`).

Reply:
Done!